multiqc/modules/picard/QualityByCycleMetrics.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+""" MultiQC submodule to parse output from Picard MeanQualityByCycle"""
+
+import logging
+import os
+import re
+
+from multiqc.plots import linegraph
+from .util import read_histogram
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+def parse_reports(self):
+    """ Find Picard QualityByCycleMetrics reports and parse their data """
+
+    headers = ['CYCLE', 'MEAN_QUALITY']
+    formats = [int, float]
+    all_data = read_histogram(self, 'picard/quality_by_cycle', 'MeanQualityByCycle', headers, formats)
+
+    if not all_data:
+        return 0
+
+    # Write parsed data to a file
+    self.write_data_file(all_data, 'multiqc_picard_quality_by_cycle')
+
+    # Plot the data and add section
+    pconfig = {
+        'id': 'picard_quality_by_cycle',
+        'title': 'Picard: Mean Base Quality by Cycle',
+        'ylab': 'Mean Base Quality',
+        'xlab': 'Cycle #',
+        'xDecimals': False,
+        'tt_label': '<b>cycle {point.x}</b>: {point.y:.2f} %',
+        'ymax': 100,
```

Comment:
As above, from reading the Picard / GATK docs I think it sounds like this is not a percentage?

Reply:
copy-paste error